REMARKs/SolvingMicroDSOPs.md

```diff
@@ -4,45 +4,37 @@ tags:
   - Replication
   - Teaching
   - Tutorial
----
-# SolvingMicroDSOPs Is an Exploration of Indirect Inference for a Life Cycle Model
 abstract: "These notes describe tools for solving microeconomic dynamic stochastic optimization problems, and show how to use those tools for efficiently estimating a standard life cycle consumption/saving model using microeconomic data.  No attempt is made at a systematic overview of the many possible technical choices; instead, I present a specific set of methods that have proven useful in my own work (and explain why other popular methods, such as value function iteration, are a bad idea).  Paired with these notes is Mathematica, Matlab, and Python software that solves the problems described in the text." # abstract: optional
 authors: # required
   -
     family-names: Carroll
     given-names: "Christopher D."
     orcid: "https://orcid.org/0000-0003-3732-9312"
+  -
+    family-names: Wang
+    given-names: Tao
+    orcid: "https://orcid.org/0000-0003-4806-8592"
+    
 cff-version: "1.1.0" # required 
-date-released: 2022-02-20 # required
+date-released: 2021-02-20 # required
 identifiers: # optional
   - 
     type: url
-    value: "https://github.com/econ-ark/SolvingMicroDSOPs"
-  - 
-    type: doi
-    value: "10.5281/zenodo.1234"
+    value: "https://llorracc.github.io/SolvingMicroDSOPs"
+
 keywords: # optional
   - Consumption
-message: "The tools " # required
+  - Saving 
+message: "" # required
 repository-code: "https://github.com/econ-ark/SolvingMicroDSOPs" # optional
 title: "Solution Methods for Microeconomic Dynamic Stochastic Optimization Problems" # required
 version: "1.0" # required
 # REMARK fields
-github_repo_url: https://github.com/econ-ark/SolvingMicroDSOPs # required 
+github_repo_url: https://github.com/econ-ark/EpiExp # required 
 commit: # Git commit number that the REMARK will always use; required for "frozen" remarks, optional for "draft" remarks
 remark-name: SolvingMicroDSOPs # required 
-title-original-paper: Solution Methods for Microeconomic Dynamic Stochastic Optimization Problems # optional 
 notebooks: # path to any notebooks within the repo - optional
-  - 
-    SolvingMicroDSOPs.ipynb
-dashboards: # path to any dahsboards within the repo - optional
-   - 
-      type: url 
-      value: https://llorracc.github.io/SolvingMicroDSOPs
-   - 
-      type: doi
-      value: 
+  - SolvingMicroDSOPs.ipynb
```

Comment:
I think this needs to be the path to the notebook -- isn't it in Code/Python/SolvingMicroDSOPs.ipynb or something like that?  Please look at some other examples.

Reply:
I did it in that way simply EpiExp has no path, either. I checked a few other REMAKRs, the format of this entry seems to differ from one to another. For instance, BufferStockTheory has an empty entry. BlanchardPA2019 has an entry like "Code/Python/BlanchardPA2019.ipynb". 

I might be wrong, but it seems that this is simply a Metadata information shown up in the Econ-ark/REMARK page, but it does not link to any original notebook.  

Is there a consistent way of doing so?